src/saving/InProgressLoad.cs

```diff
@@ -41,6 +42,11 @@ private enum State
         Finished,
     }
 
+    public static bool CheckIsLoading()
```

Comment:
You should make this a static property instead, that way the backing field is also not needed as long as the setter is private.